extensions/resource-deployment/src/ui/resourceTypeWizard.ts

```diff
@@ -58,13 +72,19 @@ export class ResourceTypeWizard {
 			return new NotebookWizardModel(this.provider, this);
 		} else if (instanceOfAzureSQLDBDeploymentProvider(this.provider)) {
 			return new DeployAzureSQLDBWizardModel(this.provider, this);
+		} else {
+			return new PageLessDeploymentModel(this.provider, this);
 		}
-		// other types are undefined for now.
-		return undefined;
 	}
 
 	public async open(): Promise<void> {
-		this.model.initialize();
+		this.model = this.getResourceProviderModel();
```

Comment:
model is update here as well as by getting the set Model setter. When is each used?